src/types.ts

```diff
@@ -2113,6 +2115,7 @@ export interface WriteCallFunctionDto {
     feeLevel?: FeeLevel;
```

Comment:
Looks like 'fee' is missing here. Let's add that.

Reply:
added